MimeKit/ContentType.cs

```diff
@@ -231,9 +231,7 @@ public ContentType (string mediaType, string mediaSubtype)
 		/// Gets the simple mime-type.
 		/// </remarks>
 		/// <value>The mime-type.</value>
-		public string MimeType {
-			get { return string.Format ("{0}/{1}", MediaType, MediaSubtype); }
-		}
+		public string MimeType => $"{MediaType}/{MediaSubtype}";
 
```

Comment:
I would definitely take the `$"{MediaType}/{MediaSubtype}"` over string.Format().